templates/tfengine/components/cicd/configs/tf-plan.yaml

```diff
@@ -33,7 +33,8 @@ steps:
 
   # Check for delete operations as an FYI, it won't fail the build.
   - name: "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:a09ad46f513847d31e2cfe20f818c2d64809e7d12695e2e6098900296f6527bf"
-    entrypoint: '/workspace/${_TERRAFORM_ROOT}/cicd/configs/tf-deletion-check.sh'
+    entrypoint: bash
+    args: ["../cicd/configs/tf-deletion-check.sh"]
```

Comment:
I think this can be replaced with `${_TERRAFORM_ORG}` rather than a relative path.

Reply:
Specifically the ".."

Reply:
Because the dir is ${_TERRAFORM_ORG}/org and I want to avoid hardcoding the /workspace.

Reply:
Nvm. This requires a full path (so /workspace), so I think it's actually better to use relative path here (since if "workspace" ever gets renamed, this script won't break since it's inserted automatically when used in "dir:").

Reply:
exactly